book/src/external-snapshotter.md

```diff
@@ -10,7 +10,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version
 --|--|--|--|--|--|--|--
-[external-snapshotter v2.0.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v2.0.1) | [release-2.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-2.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-snapshotter:v2.0.1 | v1.17 | - | v1.17
+[external-snapshotter v2.1.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v2.1.0) | [release-2.1](https://github.com/kubernetes-csi/external-snapshotter/tree/release-2.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-snapshotter:v2.1.0 | v1.17 | - | v1.17
```

Comment:
Should the "recommended k8s version" be renamed to "recommended min  k8s version" instead?  I don't see any reason why it works better in 1.17 than 1.18?

Reply:
Yes "recommended min version" is more accurate. Or maybe "fully-featured min version"?

Reply:
"fully-featured" sounds a little weird:).  Maybe "recommended min version" is better.

Reply:
Thinking about it some more, I think saying recommended min gets tricky around alpha features which may break in future releases.

Reply:
That also applies to "recommended version"?

Reply:
I guess if we say 1.17 is the recommended version, does that suggest we have not tested with 1.18 or something does not work with 1.18?

Reply:
It's tricky. We test the recommended version+ until we release another sidecar version with a higher recommended version.

For example, external-attacher 2.0 has recommended 1.15. So we run tests using 2.0 on 1.15 and 1.16.  But 2.1 recommended is 1.17. So we run tests using 2.1 using 1.17 (and 1.18)

Reply:
Actually...... I take that back.

We do test all future K8s versions. The hostpath-driver deployment X reflects the recommend versions.

And then we do skew testing for all future releases, ie deploy 1.16 on k8s 1.16, 1.17, 1.18.  deploy 1.17 on k8s 1.17, 1.18. deploy 1.18 on k8s 1.18

Reply:
https://k8s-testgrid.appspot.com/sig-storage-csi-ci